src/app/public/modules/confirm/confirm-config.ts

```diff
@@ -7,9 +7,28 @@ import {
 } from './confirm-button-config';
 
 export interface SkyConfirmConfig {
+  /**
+   * Specifies the message to display in a bold font at the top of the dialog.
+   * @required
+   */
   message: string;
+  /**
+   * Specifies secondary text to display in plain font under the primary message.
+   */
   body?: string;
+  /**
+   * Specifies an array of `SkyConfirmButtonConfig` objects that overwrite the default
+   * configuration for buttons. The number of items in the array must match the number of
+   * buttons that the `type` property specifies.
+   */
   buttons?: SkyConfirmButtonConfig[];
+  /**
+   * Indicates whether to preserve whitespace and new lines inside the confirm component.
+   * @default false
```

Comment:
We've included the `@default` parameter, but it doesn't seem to be displaying. @Blackbaud-SteveBrush, do we need to do something different, or should we file an issue in the `docs-tools` repo?